### ISSUE TITLE
GNOME 45 support

### ISSUE THREAD

**Issue by yukina3230** [mrEDitor/gnome-shell-extension-scroll-panel#30]
Hi @mrEDitor , GNOME 45 beta has been released, could you port this to the new version, porting may need some changes.

This link might help: https://gjs.guide/extensions/upgrading/gnome-shell-45.html#esm

Thank you.

**Comment by Maciejka1** [comment]
Please fix it as fast as you can ://

**Comment by mrEDitor** [comment]
Yep, sorry for inconvenience, but it seems that Gnome 45 hidden some APIs and I didn't manage to make the extension fully operational yet. Currently I'm looking into possibility to make core version of the extension to support main feature only, i.e. switching workspaces or windows, but without proper UI, namely preferences and switching visualization. Hope to provide some update this weekend.

**Comment by unicornsandcandies** [comment]
> Yep, sorry for inconvenience, but it seems that Gnome 45 hidden some APIs and I didn't manage to make the extension fully operational yet. Currently I'm looking into possibility to make core version of the extension to support main feature only, i.e. switching workspaces or windows, but without proper UI, namely preferences and switching visualization. Hope to provide some update this weekend.

any updates?
